generated/nimodinst/session.py

```diff
@@ -165,42 +166,177 @@ def close(self):
     ''' These are code-generated '''
 
     def _close_installed_devices_session(self, handle):
+        '''_close_installed_devices_session
```

Comment:
What's up with this change? Is this an independent thing?

Reply:
I just copied the part of the template from build/templates/session.py.mako that generated the method. Apparently we weren't adding the docstring before.

Reply:
Shouldn't this be a separate PR?